### PR TITLE
High priority code fixes + Few Minor code Fixes

### PR DESCRIPTION
**Pull request by HasnatL-Adapptlimited**
# Links
[Base PR](https://github.com/ReliefApplications/ems-frontend/pull/2460)

# High Priority Bugs
- [x] Reverted Alert class to prevent leak
- [ ] Style Sheet imported in kendo css class

# Other Bugs
- [ ] Removed console logs
- [ ]  Seperate changed to Separate Everywhere
- [ ] Added some missing global variable comments

### CODE REVIEW

**Comment by HasnatL-Adapptlimited** [comment]
Hey @AntoineRelief and @Matthis-M-ReliefApps
I have opened this PR, for all the high priority code fixes

File: libs/shared/src/lib/components/email/email.service.ts [hunk 1]
```diff
@@ -18,6 +18,10 @@ import { RestService } from '../../services/rest/rest.service';
   providedIn: 'root',
 })
 export class EmailService {
+  /** INDEX OF CURRENT DATASET BLOCK */
```
Comment (AntoineRelief) [inline libs/shared/src/lib/components/email/email.service.ts:21]: @HasnatL-Adapptlimited @morgang-adappt 
I haven't see that previously, but avoid capitalized comments

I'll correct that in this PR, no need you edit it

File: libs/styles/src/lib/kendo/theme.scss [hunk 2]
```diff
@@ -1,4 +1,5 @@
 @import 'variables.scss';
+@import url('https://unpkg.com/@progress/kendo-theme-bootstrap@latest/dist/all.css');
```
Comment (AntoineRelief) [inline libs/styles/src/lib/kendo/theme.scss:2]: @HasnatL-Adapptlimited 

do you need this for anything else than the kendo-stepper & kendo-progress bar?

File: libs/shared/src/lib/components/email/email.service.ts (hunk 1 above)
Comment (HasnatL-Adapptlimited) [reply libs/shared/src/lib/components/email/email.service.ts:21]: Will correct any capitalised comments I see in the next coming PR's

**Comment by AntoineRelief** [comment]
@HasnatL-Adapptlimited 

merged

cc @morgang-adappt

File: libs/styles/src/lib/kendo/theme.scss (hunk 2 above)
Comment (HasnatL-Adapptlimited) [reply libs/styles/src/lib/kendo/theme.scss:2]: I will double check but I don't believe so

**Pull request merged**